src/components/unsigned-sweep.js

```diff
@@ -45,10 +50,31 @@ class UnsignedSweep extends Component {
     btg: ['userKey', 'userKeyID', 'backupKey', 'backupKeyID', 'bitgoKey', 'recoveryDestination', 'scan'],
     zec: ['userKey', 'userKeyID', 'backupKey', 'backupKeyID', 'bitgoKey', 'recoveryDestination', 'scan'],
     dash: ['userKey', 'userKeyID', 'backupKey', 'backupKeyID', 'bitgoKey', 'recoveryDestination', 'scan'],
-    eth: ['userKey', 'userKeyID', 'backupKey', 'backupKeyID', 'walletContractAddress', 'recoveryDestination', 'apiKey', 'gasLimit', 'gasPrice'],
+    eth: [
+      'userKey',
+      'userKeyID',
+      'backupKey',
+      'backupKeyID',
+      'walletContractAddress',
+      'walletPassphrase',
```

Comment:
What is the reason behind adding walletPassphrase in unsigned recovery? I think this field will not be needed in unsigned recovery as the transaction is supposed to be signed offline